src/elements/content-uploader/removeCellRenderer.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @flow
+ * @file Function to render the remove table cell
+ */
+
+import React from 'react';
+import ItemRemove from './ItemRemove';
+import { STATUS_ERROR, STATUS_IN_PROGRESS } from '../../constants';
+
+type Props = {
+    rowData: UploadItem,
+};
+
+export default (onClick: Function) => ({ rowData }: Props) => {
```

Comment:
Can we add more specific types for all new code? We generally try to avoid Function/Object/any when possible.